tools/integration_tests/util/mounting/dynamic_mounting/dynamic_mounting.go

```diff
@@ -60,6 +60,7 @@ func runTestsOnGivenMountedTestBucket(bucketName string, flags [][]string, rootM
 
 		setup.SetMntDir(mntDirOfTestBucket)
 
+		log.Printf("Running tests with flags: %s", flags[i])
```

Comment:
Can we add > Running `dynamic mounting` tests with flag.. and similar at other places

Reply:
On the top we are already printing the same (e.g. line#119). Should we keep that also then?

Reply:
I think it will make it easier to search in case of issues.